### DISCUSSION TITLE
Kutular ve ödül sorusu (Monty)

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#86] [Zeka Sorusu]
Bir yarışmadasınız, sunucu size 3 kutu gösterdi ve birini seçmenizi istedi. Kutulardan birinde ödül var diğerleri boş. Sunucu hangi kutuda ödül olduğunu biliyor. Seçiminizi yaptıktan sonra sunucu geri kalan kutulardan boş olduğunu bildiği bir kutuyu açıyor. Sonra “kutunuzu diğer kapalı kutuyla değiştirmek ister misiniz?” diye soruyor. Değiştirmek mi değiştirmemek mi kazanma olasılığınızı arttırır.

**Comment by Berkek007** [comment]
İki durum vardır

1.Kutuyu değiştirmek istiyorum
2.Kutuyu değiştirmek istemiyorum

Durum 1 için, Eğer değiştireceğimizi biliyorsak, ödülü kazanmak için boş olan bir kutuyu seçmemiz gerekir. Boş olan bir kutuyu seçtiğimiz için, sunucu yalnızca başka bir boş olan kutuyu açmalıdır, böylece kalan kutuda değiştirerek elde ettiğimiz bir ödül olur.

Yani, boş olan bir kutuyu seçme olasılığı 2/3'tür çünkü 3 kutudan 2 si boştur. Bu nedenle, değiştirerek bir ödülü kazanma olasılığı 2/3'tür.

Durum 2  için, Eğer değiştirmeyeceğimizi biliyorsak, ödülü kazanmak için içinde ödül olan bir kutuyu seçmemiz gerekiyor.

Yani 3 kutudan 1'inde ödül olduğu için içinde ödül olan bir kutuyu seçme olasılığı 1/3'tür. Bu nedenle değiştirmeyerek ödül kazanma olasılığı 1/3'tür.

Değiştirerek ödül kazanma olasılığı değiştirmeme olasılığından daha yüksek olduğundan değiştirmek avantajlıdır.

**Comment by Sevvalgocgun1** [comment]
Sorunun Çözümü:

1. İlk başta 3 kutudan birini seçiyorsunuz. Bir kutuda ödül var, diğer ikisi boş. İlk seçimde ödül olan kutuyu seçme olasılığınız 1/3

Değiştirirsek ne olur

Eğer kutunuzu değiştirirseniz, başlangıçta ödül olmayan kutulardan birini seçmiş olma olasılığınız 2/3 olur, çünkü ilk başta doğru kutuyu seçme olasılığınız sadece 1/3'tü. Yani, başlangıçta doğru kutuyu seçmediyseniz, değiştirmek size kazandıracaktır.

Değiştirmezseniz Ne Olur?

İlk başta seçtiğiniz kutuda ödül olma olasılığınız 1/3. Bu durumda, değiştirmemek ödülü kazanmanızı sağlar, ama bu ihtimal daha düşüktür.

Sonuç:

Kutunuzu değiştirirseniz, kazanma olasılığınız 2/3 olur. Değiştirmezseniz ise 1/3 olur. Bu nedenle, kutunuzu değiştirmek kazanma olasılığınızı artırır.
